deny.toml

```diff
@@ -59,10 +59,12 @@ skip = [
     # Since `syn` is used by proc-macros (executed at compile time), duplicate
     # versions won't have an impact on the final binary size.
     { name = "syn" },
-]
-skip-tree = [
-    # Hasn't seen a new release since 2017. Pulls in an older version of nom.
-    { name = "procinfo" },
+    # right now we have a mix of versions of this crate in the ecosystem
+    # procfs uses 0.36.14, tempfile uses 0.37.4
+    { name = "rustix" },
```

Comment:
i think if we moved this from a `skip` section to a `skip_tree` section, we wouldn't need the separate entry for `linux-raw-sys`.